README.md

```diff
@@ -42,7 +42,7 @@ More information about building GoogleTest can be found at
 
 ## Features
 
-*   An [xUnit](https://en.wikipedia.org/wiki/XUnit) test framework.
+*   A [xUnit](https://en.wikipedia.org/wiki/XUnit) test framework.
```

Comment:
I don't think this change is correct. "An" is used when a word begins with a vowel sound, and here is begins with an "ex" sound. Think "an xray".